app/services/jira_client/repository.rb

```diff
@@ -36,5 +36,9 @@ def fetch_response
         request.params = @request_params
       end
     end
+
+    def raised_exception
+      raise JiraBoards::NoProjectKeyError, @jira_board.jira_project_key
```

Comment:
before this change we were rescuing an error and tracking it in exception hunter. Now we are rescuing an error and raising another error. We are changing behavior (e.g. this will probably be retried by activejob). Is that on purpose?

Reply:
Isn't this the way to handle custom exceptions like https://github.com/rootstrap/rs-code-review-metrics/tree/develop/app/exceptions as you suggested? If not the case, I think that tracking ExceptionHunter is fine. WDYT?

Reply:
there are 2 different things:
1. using `StandardError` vs a custom exception like `JiraBoards::NoProjectKeyError` -> let's use `JiraBoards::NoProjectKeyError` 👍 
2. raising the error vs calling `ExceptionHunter.track` -> let's call `ExceptionHunter.track`

Reply:
I think is done, I've got confused 😅